rholang/src/test/scala/coop/rchain/rholang/interpreter/NormalizeTest.scala

```diff
@@ -178,6 +210,9 @@ class CollectMatcherSpec extends FlatSpec with Matchers {
     )
     result.knownFree should be(inputs.knownFree.newBindings(newBindings)._1)
   }
+  "Map" should "sort elements" in {
+    assertEqualNormalized("@0!({0 : {1 | 2}})", "@0!({0 : {2 | 1}})")
```

Comment:
please also make the map keys a non-trivial par, or add a separate assert for that

Reply:
Done